solutions/18.py

```diff
@@ -0,0 +1,20 @@
+#Himanshu Nachane (@Himanshu1495)
+'''return missing number in an input array of consecutively increasing numbers.
+
+i: [5,6,8,9]
+o: 7
+'''
+
+def missing_num(arr):
+	#get first element of array
+	start = arr[0]
+	#get last element of array
+	end = arr[-1]
+	#create new array by iterating from start element to end element
+	new_arr = [x for x in range(start,end+1)]

```

Comment:
@songz , array has been used with a purpose of using a built-in function **sum(name_of_array)** Here is the line which uses **sum()** ,`    return sum(new_arr)-sum(arr)` . We can use a for loop to do the same task but in my opinion, **sum()** looks cleaner. If you insist on using a loop, I will certainly be able to do that too? Should I use loop or keep it as it is ?